content/04.methods.md

```diff
@@ -112,7 +112,7 @@ The results from both these methods are available in the filtered and processed
 10x Genomics' Space Ranger [@url:https://www.10xgenomics.com/support/software/space-ranger/latest] was used to quantify gene expression data from spatial transcriptomics libraries.
 `cellranger mkref` was used to create a reference index from the human genome, GRCh38, Ensembl version 104.
 The FASTQ files, microscopic slide image, and slide serial number were provided as input to `spaceranger count`.
-The raw and filtered counts matrix and the summary report output by `spaceranger count` are included in the folder output from `scpca-nf`.
+The raw and filtered counts matrix, slide images, and the summary report output by `spaceranger count` are included in the folder output from `scpca-nf`.
```

Comment:
This seems to be the only place we use the phrase "folder output". Maybe we should just say output and leave out "folder" as an extraneous detail.